jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/jpa/ContextMappingInfo.java

```diff
@@ -44,19 +44,15 @@
                                         @Index(name = "IDX_CMI_Owner", columnList = "OWNER_ID")})
 @SequenceGenerator(name="contextMappingInfoIdSeq", sequenceName="CONTEXT_MAPPING_INFO_ID_SEQ")
 @NamedQueries(value=
-    {@NamedQuery(name="FindContextMapingByContextId", 
-                query="from ContextMappingInfo where contextId = :contextId"
-                		+ " and ownerId = :ownerId"),
-                @NamedQuery(name="FindContextMapingByKSessionId", 
-                query="from ContextMappingInfo where ksessionId = :ksessionId"
-                		+ " and ownerId = :ownerId"),
-                @NamedQuery(name="FindKSessionToInit", 
-                query="select cmInfo.ksessionId from ContextMappingInfo cmInfo, "
+    {@NamedQuery(name="FindContextMapingByContextId", query="from ContextMappingInfo where contextId = :contextId and ownerId = :ownerId"),
+     @NamedQuery(name="FindContextMapingByAuditContextId", query="select o from ContextMappingInfo o join ProcessInstanceLog log on log.correlationKey = o.contextId where CONCAT(log.processInstanceId, '') = :contextId and o.ownerId = :ownerId"),
```

Comment:
Is there another way to convert _log.processInstanceId_ to String not using CONCAT function?

Reply:
that could work in every db, no.